wazo_calld/plugins/voicemails/exceptions.py

```diff
@@ -101,3 +101,15 @@ def __init__(self, greeting):
                 'greeting': greeting,
             }
         )
+
+
+class VoicemailGreetingAlreadyExists(APIException):
+    def __init__(self, greeting):
+        super().__init__(
+            status_code=409,
+            message='Voicemail greeting already exists',
+            error_id='voicemail-greeting-already-exists',
+            details={
+                'greeting': greeting,
+            }
+        )
```

Comment:
I think we should also set a key `"voicemail_id": voicemail_id` in `details, but we can wait until we need it